app/counters/workflow_counter.rb

```diff
@@ -7,10 +7,6 @@ def initialize(workflow)
   end
 
   def classifications
-    scope = workflow.classifications
-    if launch_date = workflow.project.launch_date
-      scope = workflow.classifications.where("created_at >= ?", launch_date)
-    end
-    scope.count
+    SubjectWorkflowStatus.where(workflow: workflow).sum(:classifications_count)
```

Comment:
I'm fine with including those in the counts (due to speed, and because the classifications happened) but it does keep confusing people.